### PR TITLE
First bunch of changes for v4

### PR DESCRIPTION
**Pull request by alexmamonchik**
- update all possible gems
- merge some PRs with "will be in v4" label
- fix failed tests

### CODE REVIEW

**Comment by dja** [comment]
Any update on this?

**Comment by alexmamonchik** [comment]
> Any update on this?

Actually, I did nothing for the last 2 months here. What do you think if we can release v4 as is now?

File: README.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ version `3.0.0` has [breaking changes](https://github.com/stripe-ruby-mock/strip
 
 ### Requirements
 
-* ruby >= 2.6.0
-* stripe >= 5.0.0
+* ruby >= 2.7.0
+* stripe > 5 & < 11
```
Comment (stevenharman) [inline README.md:33]: Stripe SDK version 11 is now out.

Comment (stevenharman) [reply README.md:33]: I just opened #905 to open this constraint, allowing Stripe v11 too.

**Comment by yaroslav-mo** [comment]
+1 to this. The Stripe gem is already v11.

**Pull request merged**